aicssegmentation/structure_wrapper/WorkflowEngine.py

```diff
@@ -176,3 +192,19 @@ def get_thumbnails(self, type="pre") -> np.ndarray:
         # TODO: need to save image in format workflowName_type.tif
         return np.squeeze(imread(os.path.join(self.workflow_name, "_", type, ".tiff")))
 
+    def get_all_params(self) -> List[Dict[str, Any]]:
```

Comment:
What is the thought behind this? When / where would we need a big list of all available parameters without their association to a step?

Reply:
Made this before I started widget generation for each step- now I think we can get rid of this since it only belongs in the scope of WorkflowStep